### DISCUSSION TITLE
IfcConvert webside download links does not work

### DISCUSSION THREAD

**Discussion by kentzversky** [IfcOpenShell/IfcOpenShell#5071] [General]
I'm trying to download ifcConvert from main website:
https://docs.ifcopenshell.org/ifcconvert/installation.html

None of the pre-built package links seems to work. I'm receiving a message:
![Screenshot 2024-07-24 095015](https://github.com/user-attachments/assets/4836badc-6ff3-4e70-877e-433ad3c0afea)

Can anyone tell me where I can download ifcConvert?
Thanks

**Comment by boroczkigabor** [comment]
Same here, could anyone please update the links and/or reupload the missing files?

**Comment by boroczkigabor** [comment]
Found out that IfcOpenBot builds them, e.g. on latest commit:
https://github.com/IfcOpenBot/IfcOpenShell/commit/d51fa2c5f77fa5494302a77f2f19b04fce4e9855#comments

@kentzversky FYA

**Comment by Moult** [comment]
Thanks for the heads up - been changing version strings and deploy methods and now things should be OK again.

**Comment by kentzversky** [comment]
Thanks for updating.

**Comment by Woopsalla** [comment]
The links are still broken.
